CONTRIBUTING.md

```diff
@@ -286,10 +286,11 @@ Flags:
 
 Optional commands to execute:
 
--K, --shell-access
-        Open shell access to Airflow cluster.
+-K, --shell-to-composer-worker
+        Open shell access to Airflow's worker. This allows you to test commands in the context of the Airflow instance.
+        It is worth noting that it is possible to access the database.
```

Comment:
Maybe we can name it `--ssh-composer-worker` and `--ssh-dataproc-master`? I think that usually when you ssh  you do this to use shell of the other machine so the command will be well understood.

Reply:
SSH allows for much more. We often use these functions, eg port forwarding. I think the distinction is correct.

Reply:
I agree with @nuclearpinguin that SSH is better name. Shell is bash/zsh etc and we are actually doing SSH (and in the future we might also add port forwarding if needed, and others). I think we should simply follow gcloud approach: `gcloud compute ssh` not `gcloud compute shell`

Reply:
It is not a well-based argument but when you look for "how to ssh to kubernetes pod" the first [result](https://kubernetes.io/docs/tasks/debug-application-cluster/get-shell-running-container/) was the actual base for #293.

Reply:
@potiuk We do not have access to SSH on Airflow container. We only run bash in the container.

Reply:
We discussed this topic offline.  We choose ``ssh``.